src/dbnode/integration/run_custom_backgroun_process_test.go

```diff
@@ -0,0 +1,93 @@
+// +build integration
```

Comment:
nit: Update the file name to use `background`

Reply:
👍